data/remote_control_interface_web/js/script.js

```
@@ -0,0 +1,275 @@
+const joystickState = {
+    use_web_inputs: true,
+    left: { x: 0.0, y: 0.0 },
+    right: { x: 0.0, y: 0.0 },
+    dead_zone_left: { x: 0.1, y: 0.1 },
+    dead_zone_right: { x: 0.1, y: 0.1 },
+    scale_left: { x: 1.0, y: 1.0 },
+    scale_right: { x: 1.0, y: 1.0 }
+};
+
+const lastSentState = JSON.parse(JSON.stringify(joystickState));
+let dirtyKeys = new Set();
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'dirtyKeys' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)